### PR TITLE
Add conda-forge badge in README.md, format as table

### PR DESCRIPTION
**Pull request by echedey-ls**
Preview at https://github.com/echedey-ls/SciencePlots/tree/update-readme-md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,38 @@
 Science Plots
 =============
 
+<p align="left">
```
Comment (Borda) [inline README.md:4]: why not keep simple MD?

Comment (echedey-ls) [reply README.md:4]: I think all MD tables render a header, even if left blank.
https://stackoverflow.com/questions/17536216/create-a-table-without-a-header-in-markdown

Thanks for the review.

Comment (Borda) [reply README.md:4]: sure but MD is simple on purpose so not sure if adding HTML is the right way...

Comment (echedey-ls) [reply README.md:4]: HTML is well known IMO. In any case, in the future it can be changed.

**Pull request merged**